api4/license_test.go

```diff
@@ -15,22 +17,22 @@ func TestGetOldClientLicense(t *testing.T) {
 	license, resp := Client.GetOldClientLicense("")
 	CheckNoError(t, resp)
 
-	if len(license["IsLicensed"]) == 0 {
-		t.Fatal("license not returned correctly")
-	}
+	require.NotEqual(t, license["IsLicensed"], 0, "license not returned correctly")
 
 	Client.Logout()
 
 	_, resp = Client.GetOldClientLicense("")
 	CheckNoError(t, resp)
 
-	if _, err := Client.DoApiGet("/license/client", ""); err == nil || err.StatusCode != http.StatusNotImplemented {
-		t.Fatal("should have errored with 501")
-	}
+	_, err := Client.DoApiGet("/license/client", "")
+	require.Error(t, err, "get /license/client did not return an error")
+	require.Equalf(t, err.StatusCode, http.StatusNotImplemented,
+		"expected 501 Not Implemented, got %v", err.StatusCode)
```

Comment:
You don't need to log the status code. testify will do this for you.

Reply:
Good to know, thanks; fixed.